tools/spread-shellcheck.py

```diff
@@ -0,0 +1,286 @@
+#!/snap/core/current/usr/bin/python3
```

Comment:
```suggestion
#!/usr/bin/env python3
```
In our case, we do not always have core installed.

Reply:
hmmm. I changed `.travis.yml` to install shellcheck from the snap, as the one in travis (and whatever you were using locally, apparently) had trouble when run via `xargs` (note how it didn't catch everything). So now `run_static_tests` has
```sh
find . ... blah | blah ... | xargs /snap/bin/shellcheck
```
if that change is ok, then this one would be ok as well :-)

of course the reason it's like this is because _this_ `spread-shellcheck` is copied from the snap I was preparing, which is classic but uses python3 from core...

but, you tell me.